themes/default/content/blog/pulumi-esc-public-preview-for-1password-support/index.md

```diff
@@ -0,0 +1,118 @@
+    ```
+
+4. Ms. Code successfully authenticates with her Google OAuth credentials.
+![Screenshot of successfully obtaining the Google User details](example-2.png)
+5. At a later point, Ms. Code needs DockerHub credentials to push her image, so she asks Mr. Keys to add these. Mr. Keys stores the Docker username and personal access token in 1Password. (This part is not shown in the demo clip.)
+![Screenshot of DockerHub creds stored in 1Password](example-3.png)
+6. Mr. Keys updates the Pulumi ESC Environment, `​​buzz-dev-environment`
+
+    ```yaml
+    values:
+      1password:
+        secrets:
+          fn::open::1password-secrets:
+            login:
+              serviceAccountToken:
+                fn::secret:
+                  ciphertext: ZXN*not shown*...=
+            get:
+              google_oauth_client_id:
+                ref: "op://dev-environment/buzz/username"
+              google_oauth_client_secret:
+                ref: "op://dev-environment/buzz/credential"
+              docker_pat:
+                ref: "op://dev-environment/dockerhub/password"
+              docker_usr:
+                ref: "op://dev-environment/dockerhub/username"
+      environmentVariables:
+        GOOGLE_OAUTH_CLIENT_ID: ${1password.secrets.google_oauth_client_id}
+        GOOGLE_OAUTH_CLIENT_SECRET: ${1password.secrets.google_oauth_client_secret}
+        DOCKER_PAT: ${1password.secrets.docker_pat}
+        DOCKER_USR: ${1password.secrets.docker_usr}
+    ```
+
+7. Lastly, Ms. Code then consumes the credentials via Pulumi ESC to log in and push the image
+
+    ```bash
+    $ esc run buzz-dev-environment  -- bash -c 'echo "$DOCKER_PAT" | docker login -u $DOCKER_USR --password-stdin'
+    Login Succeeded
+
+    $ TAG="nullstring/buzz:dev"
+    $ docker build . -t $TAG
+    $ docker push $TAG
+    ```
+
+## Summary
+
+By joining forces with 1Password, Pulumi ESC strives to offer a "secrets agnostic" tool that effortlessly integrates with various secrets management solutions. This integration is poised to become an essential component in developers' toolkits, allowing for an easy-to-use developer experience while elevating security standards in their cloud infrastructure management.
+
+* Visit the [1Password+Pulumi ESC docs](https://www.pulumi.com/docs/esc/providers/1password-secrets/) to learn more about the syntax.
+* Visit the [Pulumi ESC docs](https://www.pulumi.com/docs/esc/environments/ ) to learn more about Pulumi ESC and a complete list of supported providers.
+
+## Next Steps
+
+Stay tuned as we evolve Pulumi ESC, ensuring it remains at the forefront of platform engineering innovation. To further enrich your skills and knowledge, we invite you to register for our upcoming workshop, ["Managing team secrets with 1Password & Pulumi ESC"](https://www.pulumi.com/resources/managing-team-secrets-1password-pulumi-esc/), designed to help you start with Pulumi ESC and its integration with 1Password.
```

Comment:
Suggest moving the workshop CTA to a "note" nearer the top.

Suggest the conclusion:
1. Is stronger on overall progress since ESC launch
2. Re-states core value proposition of ESC + 1Password
3. Provides links to docs for ESC and 1Password integration.

Reply:
We merged the suggestions.